apps/files/src/components/Trashbin.vue

```diff
@@ -40,7 +40,12 @@
         </template>
         <template #rowColumns="{ item }">
           <div class="uk-width-expand">
-            <file-item :key="item.viewId" :item="item" :name="$_ocTrashbin_fileName(item)" />
+            <file-item
+              :key="item.viewId"
+              :item="item"
+              :name="$_ocTrashbin_fileName(item)"
+              :display-preview="false"
```

Comment:
Why hide the thumbnails within the trash bin? 🤔

Reply:
The thumbnails are not yet being displayed in trashbin at all and I do not think implementing that should be part of this PR so disabling them until it's done is good enough here IMO.

Reply:
👍